### DISCUSSION TITLE
GitHub Copilot is not available

### DISCUSSION THREAD

**Discussion by trouvailleme** [community/community#113678] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

[error] [default] Extension activation failed: "Contact Support. You are currently logged in as trouvaillezhao."

**Comment by Akash1134** [comment, marked as answer]
Hi @trouvaillezhao ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
